src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ CHANGELOG
  * Deprecated passing an array as the first argument of the `CheckboxListMapper::mapFormsToData()` method, pass `\Traversable` instead.
  * Deprecated passing an array as the second argument of the `RadioListMapper::mapDataToForms()` method, pass `\Traversable` instead.
  * Deprecated passing an array as the first argument of the `RadioListMapper::mapFormsToData()` method, pass `\Traversable` instead.
+ * Added a `choice_translation_parameters` option to `ChoiceType`
```

Comment:
should it be named `label_translation_parameters`?

Reply:
`label_translation_parameters` already exists and is used for the `label`.

This option is the same but for `choices`.